docs/source/routing/self-hosted/containerization/docker.mdx

```diff
@@ -39,13 +39,7 @@ Make sure to replace `<your-graph-ref>` and `<your-graph-api-key>` with your gra
 
 ## Enabling MCP
 
-<ExperimentalFeature>
-
-This feature is [experimental](/graphos/resources/feature-launch-stages#experimental). For more information about MCP support,  review the [MCP documentation](https://www.apollographql.com/docs/apollo-mcp-server).
-
-</ExperimentalFeature>
-
-To serve MCP requests, enable the [MCP server](https://www.apollographql.com/docs/apollo-mcp-server) using the `MCP_ENABLE` environment variable. You'll also need to export container port `5000` for HTTP Streamable connections to the MCP server, using the `-p 5000:5000` flag.
+To serve MCP requests, enable the [Apollo MCP Server](/apollo-mcp-server) using the `MCP_ENABLE` environment variable. You'll also need to export container port `5000` for Streamable HTTP connections to the MCP server, using the `-p 5000:5000` flag.
```

Comment:
@DaleSeo can you confirm this whole port thing? I didn't follow your comments too closely on the MCP side re: port we recommend